README.md

```diff
@@ -83,8 +83,13 @@ make dist
 
 Running Unit Tests
 -------------------
-In order to run unit tests, pandas and h5py must be installed.
+In order to run unit tests, you need pytest
```

Comment:
Just add `pytest` to the list. `pandas` and `h5py` are test dependencies too.

Reply:
Done.